lib/manageiq/schema/schema_dumper.rb

```diff
@@ -0,0 +1,114 @@
+module ManageIQ
+  module Schema
+    module SchemaDumper
+      METRIC_ROLLUP_TABLE_REGEXP = /(?<METRIC_TYPE>metrics?_?.*)_(?<CHILD_TABLE_NUM>\d\d)$/.freeze
+
+      def tables(stream)
+        super
+        miq_metric_table_contraints(stream)
+        miq_metric_views(stream)
+        triggers(stream)
+      end
+
+      def table(table, stream)
+        super
+        add_id_column_comment(table, stream)
+        track_miq_metric_table_inheritance(table)
```

Comment:
a little surprised that you only needed to add comments to the `id`.
If rails already takes care of comments for all columns but the `id`, does it make sense to just drop our comments on the `id` and just use what is built into rails?

Sometimes nice to just go with the flow of rails

Reply:
So, I think the reason `:id` isn't include because also the "table description" is included as part of the `create_table` method arguments _**AND**_ that method signature is usually what defines our `:id` columns.

For example, for `conversion_hosts`:

```ruby
  create_table "conversion_hosts", comment: "Conversion Hosts", force: :cascade do |t|
    t.string "name", comment: "A symbolic name for the conversion host."
    t.string "address", comment: "The IP address for the conversion host. If present, must be one of the associated resource's IP addresses."
    
    # ...

  end

  change_column_comment "conversion_hosts", "id", "The internal database ID."
```

The `comment:` that is there is for the table itself, and the `id` column is just inferred by the table definition.